debian/libsvbony/changelog

```diff
@@ -1,3 +1,23 @@
+libsv305 (1.9.6) bionic; urgency=low
```

Comment:
This should be libsvbony

Reply:
Thanks for the review.
Changelog has been corrected.

Can you make the Camera Number a Feature request as an Issue? I would like to consider it separately from this Pull Request.

By the way, if this PR is merged, what will happen to indi_sv305_ccd already installed in the user's environment ? Will it be automatically removed by apt upgrade ?